libs/utils/src/sentry_init.rs

```diff
@@ -0,0 +1,21 @@
+use sentry::ClientInitGuard;
+use std::env;
+
+#[must_use]
+pub fn init_sentry(extra_options: &[(&str, &str)]) -> Option<ClientInitGuard> {
+    let dsn = env::var("SENTRY_DSN").ok()?;
+
+    let guard = sentry::init((
+        dsn,
+        sentry::ClientOptions {
+            release: sentry::release_name!(),
+            ..Default::default()
+        },
```

Comment:
I don't think placing `release_name!` here is a good idea. Judging by the fact that it's a macro and its [implementation](https://docs.rs/sentry-core/0.29.1/src/sentry_core/macros.rs.html#18), it should be called in the target app and then passed as a parameter to `init_sentry`.

That would also eliminate the need for passing app name in `extra_options`.